src/test/functional/pages/BuzzPage.ts

```diff
@@ -10,22 +10,29 @@ export class BuzzPage extends BasePage {
     readonly photoImg: Locator
     readonly noPostParagraph: Locator
     readonly buzzPageButton: Locator
+    readonly editPostParagraph: Locator
+    readonly textFieldInPostEdit: Locator
+    readonly removePictureButton: Locator
+    readonly confirmEditedPost: Locator
   
     constructor(page: Page) {
       super(page)
-      this.sharePhotosButton = page.locator('button:has-text("Share Photos")');
+      this.sharePhotosButton = page.getByText("Share Photos");
       this.fileInput = page.locator('input[type="file"]');
       this.submitPhoto = page.locator('button.oxd-button--main:has-text("Share")');
```

Comment:
Please read about recommended locators https://playwright.dev/docs/locators
If you have a problem defining a decent locator, don't hesitate to reach me - I'll help.
In this example, I think this would work, and is much cleaner:
`page.getByRole("button", { name: "Share" })`
BTW. If button says "share" maybe it's a good idea to name locator after it - I mean like "sharePhotoButton"

Reply:
Oh - I see you have one sharePhotoButton already. So this one is basically confirmation? If so - maybe sharePhotoConfirmationButton to avoid confusion?